asmcomp/linear.ml

```diff
@@ -49,6 +49,7 @@ let has_fallthrough = function
 
 type fundecl =
   { fun_name: string;
+    fun_args: Reg.Set.t;
```

Comment:
This does not seem to be used anywhere, does it?

Reply:
I have plans for minor changes to Linear after the merge, so this is likely to change anyway.